schema/alexandria_convention/snapshot/2.0.0/alexandria_convention_schema.tsv

```diff
@@ -0,0 +1,88 @@
+attribute	required	type	array	class	ontology	ontology_root	controlled_list_entries	dependency	dependency_condition	dependent	attribute_description
+biosample_id	Yes	string									Biosample ID
+biosample_type	Yes	string		enum			"[""CellLine"",""DerivedType_Organoid"",""DerivedType_InVitroDifferentiated"",""DerivedType_InducedPluripotentStemCell"",""PrimaryBioSample"",""PrimaryBioSample_BodyFluid"",""PrimaryBioSample_CellFreeDNA"",""PrimaryBioSample_PrimaryCell"",""PrimaryBioSample_PrimaryCulture"",""PrimaryBioSample_Stool"",""PrimaryBioSample_Tissue""]"				Type of Biosample
+CellID	Yes	string									Cell ID
+disease	Yes	string	TRUE	ontology	https://www.ebi.ac.uk/ols/api/ontologies/mondo	MONDO_0000001					The disease state(s) of the individual donating the sample at the time of donation

```

Comment:
This does not reflect the changes that were made in `1.1.5`, so this would be a regression.

Reply:
addressed in `fde9df9`